stackrox/stackrox
# ROX-18212: Delete Network Graph 1.0 orphan code

## Description

Files not used elsewhere from `import` statements found before deletion in #6699
* src/Components/NetworkGraph
* src/Containers/Network

## Checklist

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui
3. `yarn start` in ui

### Manual testing

1. Visit /main/dashboard

2. Click **Network Graph** in left navigation to visit /main/network-graph

3. Continue to visit every link in left navigation to see unrelated warnings:

    * Visit /main/violations: click **90-Day Image Age** and then click **Policy**
        console warning: EnvVar ROX_NETPOL_FIELDS not found in the backend list, possibly stale?
    
    * Visit /main/compliance/deployments: click a row, click link to single page, and then click **controls** tab
        console warning: Removing a style property during rerender (borderTopColor) when a conflicting property is set (borderColor) can lead to styling bugs. To avoid this, don't mix shorthand and non-shorthand properties for the same value; instead, replace the shorthand with separate values.
        at a
    
    * Visit /main/clusters: click a row
        console warning: Failed prop type: The prop `clusterRetentionInfo` is marked as required in `ClusterSummary`, but its value is `null`.

### Integration testing

1. `yarn cypress-open` in ui/apps/platform
    * networkGraphPF/networkDeploymentSidebar.test.js
    * networkGraphPF/networkGraphPF.test.js
